src/cloudinary.js

```diff
@@ -730,6 +739,103 @@ class Cloudinary {
     return Transformation.new(this.config()).fromOptions(options).setParent(this);
   }
 
+  /**
+   *
+   * @description This function will will append a TransparentVideo element to the htmlElContainer passed to it.
+   *              TransparentVideo can either be an HTML Video tag, or an HTML Canvas Tag.
+   *              Given a publicId and a set of transformation options, the function will query the resource in the CDN
+   *              and check the 'X-Cld-Vmuxed-Alpha' Header (if it's present or not).
+   *              If present, the video returned is not natively transparent, which means it has a second video attached
+   *              with the Alpha Layer.
```

Comment:
I think the how is important for this specific case, as it involves so many moving parts and dependencies.

Headers from CDN, SeeThru script, Canvas instead of Video element and such. this is context that's hard to get simply by reading the code. 
maybe that's part of the problem we could make the code more reader-friendly.